tests/scripts/docker/run_tests.sh

```diff
@@ -47,7 +47,7 @@ fi
 
 # CASS_DRIVER_NO_EXTENSIONS is set so we don't build the Cassandra C-extensions,
 # as this can take several minutes
-docker build --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
+DOCKER_BUILDKIT=1 docker build --cache-from=elasticobservability/apm-agent-python:${1} --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
```

Comment:
`elasticobservability/apm-agent-python-testing` ?

```suggestion
DOCKER_BUILDKIT=1 docker build --cache-from=elasticobservability/apm-agent-python-testing:${1} --build-arg PYTHON_IMAGE=${1/-/:} -t apm-agent-python:${1} . # replace - with : to get the correct docker image
```